### DISCUSSION TITLE
Make changes for album art.

### DISCUSSION THREAD

**Discussion by jaystocky** [purpl3F0x/TIDAL-Discord-Rich-Presence-UNOFFICIAL#75] [Ideas]
Discord implemented an API change a little while ago which lets you use direct image URLs for Album Art, make use of it! :D

**Comment by purpl3F0x** [comment]
A link would be helpful

**Comment by Totodile0** [comment]
Would it be possible to atleast change the default picture used in discord? I wasn't able to find it in the files yet

**Comment by purpl3F0x** [comment]
Album images are now available. Cheers

**Reply by Totodile0** [reply]
Thank you so much! It works perfectly, I was really thinking of perhaps changing to spotify for a month but I really love Tidal and this has made me stay now forever :) Thank you!
